common/src/dodeca.rs

```diff
@@ -127,14 +133,29 @@ impl Vertex {
 
     /// Transform from euclidean chunk coordinates to hyperbolic node space
     pub fn chunk_to_node(self) -> na::Matrix4<f64> {
-        let origin = na::Vector4::new(0.0, 0.0, 0.0, 1.0);
-        let [a, b, c] = self.canonical_sides();
-        na::Matrix4::from_columns(&[
-            a.reflection().column(3) - origin,
-            b.reflection().column(3) - origin,
-            c.reflection().column(3) - origin,
-            origin,
-        ]) * na::Matrix4::new_scaling(0.5)
+        self.dual_to_node() * na::Matrix4::new_scaling(1.0 / Self::dual_to_chunk_factor())
```

Comment:
Leaving this comment for posterity:

Before the rebase, there was an intermediate commit where this function was defined as
```rust
let [a, b, c] = self.canonical_sides();
na::Matrix4::from_columns(&[
    a.normal() * a.normal()[3],
    b.normal() * b.normal()[3],
    c.normal() * c.normal()[3],
    math::origin(),
])
```
This still used the old convention but tried to take advantage of the normals. Since the definition got changed again, this version was omitted from history after the rebase.